components/FeaturedCarousel.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react"
+import { Badge } from "@/components/ui/badge"
+import { Button } from "@/components/ui/button"
+import { Card, CardContent } from "@/components/ui/card"
+import { SectionHeader } from "@/components/ui/section-header"
+import Provident from "@/components/assets/Provident"
+import BrownLogo from "@/components/assets/BrownLogo"
+import { ChevronLeftIcon, ChevronRightIcon, UserIcon } from "lucide-react"
+import { ProfileCard } from "@/components/ProfileCard"
+
+const featuredCarouselData = [
+  {
+    title: "PROVIDENT",
+    category: "Public Health",
+    description:
+      "A web app to support the PROVIDENT research study looking to prevent drug-related deaths in neighborhoods across Rhode Island.",
+    image: Provident,
+  },
+  {
+    title: "PROJECT 2",
+    category: "Economics",
+    description: "Hello world.",
+    image: BrownLogo,
+  },
+]
+
+export const FeaturedCarousel: React.FC = () => {
+  const [currentProjectIndex, setCurrentProjectIndex] = useState(0)
+
+  const handlePrevProject = () => {
+    setCurrentProjectIndex((prevIndex) =>
+      prevIndex === 0 ? featuredCarouselData.length - 1 : prevIndex - 1
+    )
+  }
+
+  const handleNextProject = () => {
+    setCurrentProjectIndex((prevIndex) =>
+      prevIndex === featuredCarouselData.length - 1 ? 0 : prevIndex + 1
+    )
+  }
+
+  const ImageComponent = featuredCarouselData[currentProjectIndex].image;
+
+  return (
+    <section className="w-full relative mt-[90px] flex items-center justify-center">
+      {/* Chevron Left Button */}
+      <Button
+        variant="outline_neutral"
+        size="icon"
+        iconOnly={<ChevronLeftIcon className="h-8 w-8 text-neutral-500" />}
+        className="absolute left-12 top-1/2 transform -translate-y-1/2 w-[45px] h-[45px] rounded-full bg-white border border-neutral-300 shadow-lg z-10"
+        onClick={handlePrevProject}
+      />
+
+      {/* Chevron Right Button */}
+      <Button
```

Comment:
These buttons are slightly weird? Once I click, the border seems to stay black until I click somewhere else?

Totally okay to make this an issue to resolve in a different PR

Reply:
yea they are weird! theres a button PR that needs some style overhauls! both in terms of better alignment with the figma style guide, but also refactoring how our color palette is applied across not just buttons but throughout all components where relevant...